src/main/resources/application.conf

```diff
@@ -74,6 +74,9 @@ ergo {
     # Minimal fee amount of transactions mempool accepts
     minimalFeeAmount = 1000000
 
+    # If true, the node will store all transactions, boxes and addresses.
```

Comment:
All transactions, boxes and addresses are already stored. This should be changed to:

    # If true, the node will store all transactions, boxes and addresses in an index.

Reply:
yes, my mistake